### ISSUE TITLE
Case discovery does not work for all projects after #31

### ISSUE THREAD

**Issue by lextm** [xunit/xamarinstudio.xunit#34]
Some projects lead to failure in case discovery. Will explore and see why.

**Comment by lextm** [comment]
Just found the same project failed on OS X but worked on Windows. Will further analyze. It might be related to cache.

**Comment by lextm** [comment]
Further analysis shows that for such projects, a compilation without errors can trigger the correct test cases to appear in Unit Testing panel.

Therefore, close it as won't fix.
